test/Common.cpp

```diff
@@ -120,6 +122,11 @@ void CommonOptions::validate() const
 		ConfigException,
 		"Invalid test path specified."
 	);
+	assertThrow(
+		!enforceGasTest || evmVersion() == langutil::EVMVersion::istanbul(),
```

Comment:
I think it should be `EVMVersion{}` here too.

```suggestion
		!enforceGasTest || evmVersion() == langutil::EVMVersion{},
```